### PR TITLE
add changes

### PR DESCRIPTION
**Pull request by Morcoso-daisy**
add changes

### CODE REVIEW

**Comment by Ayacocho-KylieRoss** [comment]
Hello, this Kylie, and I'll be reviewing your codes now.

File: Morcoso_Daisy/script.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        <p>Name: ${comment.name}</p>
+        <p>Comment: ${comment.text}</p>
+        <p class="comment-date">Date: ${commentDate}</p>
+    `;
+    commentsSection.appendChild(commentDiv);
+  });
+}
+
+function sortComments(order) {
+  if (order === "asc") {
```
Comment (Ayacocho-KylieRoss) [inline Morcoso_Daisy/script.js]: Please follow this guidelines:
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g294850e169b_0_1

Comment (KRAyacocho) [inline Morcoso_Daisy/script.js:64]: Please follow this guidelines:
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g294850e169b_0_1

**Comment by Ayacocho-KylieRoss** [comment]
Where is the code in the HTML for the sorting buttons?
image

**Comment by Ayacocho-KylieRoss** [comment]
There seems to have a problem with the netlify deployment.

**Comment by Ayacocho-KylieRoss** [comment]
Please fix your code for adding comments

**Comment by Morcoso-daisy** [comment]
> Where is the code in the HTML for the sorting buttons? image



> Where is the code in the HTML for the sorting buttons? image

436667365_3303959476574278_4737758159649649457_n

eto po

**Comment by Morcoso-daisy** [comment]
> Where is the code in the HTML for the sorting buttons? image

<!DOCTYPE html>
<html>
<head>
  <title>Morcoso, Daisy C.</title>
  <link rel="stylesheet" type="text/css" href="style.css" />
  <style>
    .comment-section h3 {
      text-align: center;
    }
  </style>
</head>
<body>
  <h1>Morcoso, Daisy C.</h1>

  <div class="hobbies-and-interests">
    <h2>My Interests and Hobbies</h2>
    <ul>
      <li>Sleeping</li>
      <li>Cooking</li>
      <li>Eating</li>
      <li>Playing video games</li>
      <li>Watching movies</li>
    </ul>
  </div>

  <div class="works">
    <h2>My works</h2>
    <ol>
      <li><a href="chef.html">Chef</a></li>
      <li><a href="traveler.html">Traveler</a></li>
      <li><a href="photographer.html">Photographer</a></li>
      <li><a href="digital-marketing.html">Digital Marketing</a></li>
      <li><a href="teacher.html">Teacher</a></li>
    </ol>
  </div>

  <div class="my-goals-in-life">
    <h2>My Goals in Life</h2>
    <ol>
      <li>To have a better life</li>
      <li>Travel and Adventure: Exploring new places, experiencing 
          different cultures, and seeking adventure.</li>
      <li>To Graduate</li>
      <li>Providing my parents with a secure financial future</li>
      <li>Continuously learning, improving skills, and developing as 
          an individual.</li>
      <li>Build a successful career in a field I'm passionate about.</li>
    </ol>
  </div>

  <div class="my-goals-for-myself">
    <h2>My Goals for Myself</h2>
    <ol>
      <li>Find inner peace and happiness by practicing mindfulness 
          and self-care.</li>
      <li>Become a better version of myself</li>
      <li>I can do what I want to do without being told.</li>
      <li>To have a mentally and financially stable life.</li>
      <li>Build a cozy, eco-friendly home surrounded by nature.</li>
      <li>To buy what i want by not looking at the price tag.</li>
      <li>Buying own house and car.</li>
      <li>sleep all night</li>
    </ol>
  </div>

  <div class="friendship-goals">
    <h2>Friendship Goals</h2>
    <ol>
      <li>Be a loyal and trustworthy friend.</li>
      <li>Group study and overnight with them.</li>
      <li>Foodtrip with them</li>
      <li>Going out with friends.</li>
      <li>Drinking alcohol with them while having a breakdown
          is fun.</li>
      <li>Forgive and move past misunderstandings or disagreements, 
          prioritizing the friendship over any conflicts.</li>
    </ol>
  </div>

  <div class="relationship-goals">
    <h2>Relationship Goals</h2>
    <ol>
      <li>Giving him assurance.</li>
      <li>Prioritize quality time together to nurture our relationship 
          and create lasting memories.</li>
      <li>Show appreciation and affection regularly to strengthen 
          our bond.</li>
      <li>Learn each other's love language.</li>
      <li>To have a healthy relationship.</li>
      <li>Paying attention to the small things</li>
      <li>Going for coffee or ice cream dates.</li>
      <li>Movie Marathon.</li>
      <li>Take care of a dog/cat with him.</li>
      <li>Words of Affirmation.</li>
      <li>Be faithful to him.</li>
      <li>Cuddle with him(HAAHAHAHAHAHA).</li>
      </li>
    </ol>
  </div>

  <h3>Add a comment here</h3>
  <form class="add-comment">
    <input type="text" name="name" id="name" placeholder="Enter your name" />
    <textarea name="textarea-for-comment" id="textarea_for_comment"  
     placeholder="Add your comment here"></textarea>
    <button type="button" name="comment" id="comment">Comment</button>
    <button type="button" id="sort_asc">Sort by Date (Ascending)</button>
    <button type="button" id="sort_desc">Sort by Date (Descending)</button>
  </form>

  <div class="comments-of-team-comment">
    <h3>Comments</h3>
  </div>

  <div class="oc-thoughts">
    <h2>Thoughts about online class</h2>
    <div class="reasons">
      <p>
        Conducting online classes during periods of extreme heat in
        Metro Manila not only prioritizes the health and safety of individuals
        but also addresses logistical challenges, ensures continuity of 
        education, and promotes environmental sustainability. Embracing 
        digital learning platforms offers a practical and effective way to 
        navigate weather-related disruptions while providing students with 
        access to quality education from the safety and comfort of their 
        homes. Due to the high heat index in Metro Manila, conducting online 
        classes also addresses logistical challenges related to transportation
        and commuting during extreme weather conditions. Heavy traffic, 
        coupled with the intense heat, can make commuting to and from school 
        both uncomfortable and potentially hazardous for students, faculty, 
        and staff. By transitioning to online classes, individuals can avoid 
        prolonged exposure to the heat and minimize the risks associated with
        commuting in such conditions.
      </p>
    </div>
    
    <div id="testimonials">
      <h2>Testimonials</h2>
      <div class="testimonial">
        <p>
          Reyes, John Ruiz P. Monday, April 8, 2024. I've had the pleasure of
          working with Daisy on several projects, and I'm continually impressed
          by her dedication and expertise. Her attention to detail and
          problem-solving skills are second to none, making her an invaluable
          member of any team.
        </p>
      </div>
      <div class="testimonial">
        <p>
          Bergado, Shamma S.J. Monday, April 8, 2024. Coding with my group mate
          Daisy has been a breeze, thanks to her exceptional skills and 
          seamless communication. Together, we've tackled complex challenges 
          with confidence and achieved remarkable results.
        </p>
      </div>
      <div class="testimonial">
        <p>
          Paul Monday, April 8, 2024. Teaming up with my coding group has been 
          a game-changer. Their diverse expertise and strong work ethic 
          drive our projects with precision and efficiency.
        </p>
      </div>
      <div class="testimonial">
        <p>
          Norona, Mary Nicole P. Monday, April 8, 2024. Having a creative and
          great teammate made us all do great job.
        </p>
      </div>
    </div>

    <div class="contact-container">
      <h2>Contact Me</h2>
      <form class="form">
        <input type="email" name="email" placeholder="Email Address" 
         autocomplete="off" required />
        <input type="text" name="Fullname" 
         placeholder="Fullname" 
         autocomplete="off" required />
        <input type="text" name="Subject" 
         placeholder="Subject" autocomplete="off" required />
        <input type="number" name="Age" placeholder="Age" 
         autocomplete="off" required />
        <label for="copy" class="checkbox-label">
          <input type="checkbox" id="copy" name="copy" />
          <span>Send me a copy</span>
        </label>
        <textarea rows="5" cols="60" name="message" placeholder="Enter text"
          autocomplete="off" required></textarea>
        <button type="submit" name="submit" id="submit">Submit</button>
      </form>
    </div>

    <div id="gallery">
      <h2>Gallery</h2>
      daisy lower
      hirayamanarawi
      kape
      bts
      study
      coffeee
    </div>

    <script src="./script.js"></script>
  </body>
</html>

document.addEventListener("DOMContentLoaded", function () {
  let nameInput = document.getElementById("name");
  let commentTextarea = document.getElementById("textarea_for_comment");
  let commentButton = document.getElementById("comment");

  nameInput.addEventListener("input", checkFormValidity);
  commentTextarea.addEventListener("input", checkFormValidity);
  commentButton.addEventListener("click", addComment);

  document.getElementById("sort_asc").addEventListener("click", function() {
    sortComments("asc");
  });
  document.getElementById("sort_desc").addEventListener("click", function() {
    sortComments("desc");
  });

  loadComments();
});

let comments = [];

function checkFormValidity() {
  let nameValue = document.getElementById("name").value.trim();
  let commentValue = document.getElementById("textarea_for_comment").value.trim();
  document.getElementById("comment").disabled = !(nameValue && commentValue);
}

function addComment() {
  const nameInput = document.getElementById("name").value;
  const commentInput = document.getElementById("textarea_for_comment").value;
  if (nameInput.trim() && commentInput.trim()) {
    const comment = {
      name: nameInput,
      text: commentInput,
      date: new Date().toISOString(),
    };
    comments.push(comment);
    saveComments();
    displayComments();
    document.getElementById("name").value = "";
    document.getElementById("textarea_for_comment").value = "";
  }
}

function displayComments() {
  const commentsSection = document.querySelector(".comments-of-team-comment");
  commentsSection.innerHTML = "<h3>Comments</h3>";
  comments.forEach((comment) => {
    const commentDiv = document.createElement("div");
    commentDiv.classList.add("comment");
    const commentDate = new Date(comment.date).toLocaleString();
    commentDiv.innerHTML = `
        <p>Name: ${comment.name}</p>
        <p>Comment: ${comment.text}</p>
        <p class="comment-date">Date: ${commentDate}</p>
    `;
    commentsSection.appendChild(commentDiv);
  });
}

function sortComments(order) {
  if (!asc) {
    comments.sort((a, b) => new Date(a.date) - new Date(b.date));
  } else if (!desc) {
    comments.sort((a, b) => new Date(b.date) - new Date(a.date));
  }
  displayComments();
}

function saveComments() {
  localStorage.setItem("comments", JSON.stringify(comments));
}

function loadComments() {
  const storedComments = localStorage.getItem("comments");
  if (storedComments) {
    comments = JSON.parse(storedComments);
    displayComments();
  }
}

**Pull request closed without merging**